packages/swarm-components/src/Radio.jsx

```diff
@@ -1,27 +1,61 @@
 // @flow
 import * as React from 'react';
-import Icon from './Icon';
 
 export type Props = {
-  checked: boolean,
+  /**
+   * Whether the radio button is checked.
+   */
+  checked?: boolean,
+  /**
+   * Whether the input should be interactive.
+   */
   disabled?: boolean,
+  /**
+   * A unique identifier for the input.
+   */
   id?: string,
+  /**
+   * Used to associate a group of radio buttons.
+   * Only one radio button in a group can be selected.
+   */
+  name: string,
+  /**
+   * Value of the input.
+   */
   value: string,
+  /**
+   * String label for the input. Will override any children values.
+   */
   label?: string,
-  name: string,
+  /**
+   * Use children as an alternative to the `label` prop for more complex input labels.
+   * The `label` prop will override children.
+   */
+  children: React.ReactElement<*>,
```

Comment:
It looks like this is not a required prop, but it is marked as one.

Reply:
done